### ISSUE TITLE
Move Libraries outside of Resources page

### ISSUE THREAD

**Issue by sergey-tihon** [fable-compiler/fable-compiler.github.io#125]
[List of libraries](https://fable.io/resources.html#libraries) already quite long, not full and slightly hard to navigate.

@delneg already created new list of categorised libraries - https://github.com/delneg/Fable-useful-resources

I think would be nice to merge it with official documentation. As an option it can be new `Libraries` page near the `Resources`.

@fable-compiler/documentation what do you think?

**Comment by MangelMaxime** [comment]
> I think would be nice to merge it with official documentation. As an option it can be new Libraries page near the Resources.

I agree, it would be nice to avoid having a separate repositories for that. I think it is better to have a centralised place to redirect new users.

It also simplify the work of the maintainer and avoid them the need to send PRs to X repositories.

---

Another option is to add sections under `Libraries` and configure the TOC to generate quick menu access for these elements too.

Example:

![image](https://user-images.githubusercontent.com/4760796/154764427-4f3cab76-de53-4b00-9a0e-f6d45b606807.png)

It would allow to quickly navigate inside the libraries section.
